src/index.js

```diff
@@ -8,7 +8,7 @@ export default class ConfiguredElasticClient extends EventEmitter {
   constructor(context, opts) {
     super();
     assert(opts, 'configured-elasticsearch-client must be passed arguments');
-    assert(opts.hostname, 'configured-elasticsearch-client missing hostname setting');
+    assert(opts.hostname || opts.hosts, 'configured-elasticsearch-client missing hostname setting or hosts array');
```

Comment:
@djMax The underlying ElasticSearch library already supports multiple hosts.  However this Assert blocks that functionality.  The line you pointed to I added so the host parameter would be set with the first host.  It is dirty, but I'm not sure how to get which host the client is trying to connect to or if it matters.